### DISCUSSION TITLE
Skywalking agent not working with Jboss EAP7.3.2

### DISCUSSION THREAD

**Discussion by DuraiDhyaan** [apache/skywalking#11871] [General]
Hi ,
I have configured Skywalking agent with Jboss EAP7.3.2 .Once all the configurations completed while trying to start the server receiving below Errors can someone please help on this 

[Server:] INFO 2024-01-30 11:00:33.480 main SkyWalkingAgent : Skywalking agent begin to install transformer ...
[Server:] Jan 30, 2024 11:00:36 AM java.lang.System$LoggerFinder lambda$accessProvider$0
[Server: WARNING: Failed to instantiate LoggerFinder provider; Using default.
[Server:] Exception in thread "main" java.lang.IllegalStateException: The LogManager was not properly installed (you must set the "java.util.logging.manager" system property to "org.jboss.logmanager.LogManager")
[Server:]   at org.jboss.logmanager.Logger.getLogger(Logger.java:57)
[Server:   at org.jboss.as.server@10.1.11.Final-redhat-00001//org.jboss.as.server.DomainServerMain.main(DomainServerMain.java:93)
[Server:]   at org.jboss.modules.Module.run(Module.java:352)
[Server:]   at org.jboss.modules.Module.run(Module.java:320)
[Server:   at org.jboss.modules.Main.main(Main.java:593)

**Comment by wu-sheng** [comment]
We don't have much experience in JBoss. We may need someone knows more about JBoss.

**Reply by DuraiDhyaan** [reply]
Thanks ..Please can anyone help on this ?

**Comment by DuraiDhyaan** [comment]
Hi ,
Can anyone confirm Apache-Skywalking java agent supports Jboss EAP

**Comment by AmiraAloui** [comment]
Hello , 
Can anyone confirm if the Apache SkyWalking Java agent supports JBoss EAP/Wildfly? We encountered an exception when using Wildfly.
